src/AutoMerge/types.jl

```diff
@@ -90,6 +86,7 @@ end
 function GitHubAutoMergeData(;kwargs...)
     fields = fieldnames(GitHubAutoMergeData)
     @assert Set(keys(kwargs)) == Set(fields)
+    @assert kwargs[:authorization] ∈ (:normal, :jll)
```

Comment:
I notice there are a couple other places where we now use `@assert`.

In a future version of Julia, assertions will be disabled by default. Actually, there is currently a PR open to JuliaLang/julia that will disable assertions by default.

So, in this package, I think it's best if we don't use `@assert`. Instead, we have the `always_assert` function.

If we like, we could replace the `always_assert` function with an `@always_assert` macro, if that is easier to use.

Reply:
Good point, I forgot about that